### DISCUSSION TITLE
Why I can't submit my application for the student benefit?

### DISCUSSION THREAD

**Discussion by somebody-someone** [community/community#41617] [GitHub Education]
I'm requesting a student discount and had all proof ready. When I press the "Process my application" button it reply 

>Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.,Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

It is not the problem of been reject, but I can't even upload my proof. I had filled up all my profile in English whether the one I have or the one I don't. Can anyone tell me the reason, or I'm rejected simply for being from China? I'd appreciated it.

**Comment by elsaversailles** [comment]
Hey there! Try to refresh the page, I've actually renewed my today and I encountered the same issue on uploading proof

**Reply by somebody-someone** [reply]
Well, I had been refreshing it for 3 days, but the same problem always occured.

**Reply by somebody-someone** [reply]
I don't know how much longer I have to wait.

**Reply by kw927** [reply]
same here

**Reply by somebody-someone** [reply]
I don't know how long had pasted, but now I had submitted my application and been approved, after all that.

**Comment by samus-aran** [comment]
Hi @somebody-someone 👋 

We're happy you're here. Let's see how we can assist.

It looks like that the application process is asking for you to update your GitHub profile. The verification process does cross reference the evidence that you've submitted with your GitHub username, email and profile name (as well as other aspects). Do these match in your evidence? If text is in non-english you may need to translate it to english.

**Reply by somebody-someone** [reply]
All of my profile and evidence are written in English, actually. However, some of my profile, for example the twitter account, which I don't have, is bullshit, like "I_DONT_HAVE_ONE".

**Comment by echoMERN** [comment]
![Screenshot 2024-01-24 183344](https://github.com/community/community/assets/156581333/3a5b47ab-f5e7-4d1b-91a9-0f929e719ac9)
i have been waiting since 2 hours for this process to finish :(

**Comment by GitAceHub** [comment]
Hello everyone,
I encountered the same issue while trying to upload my proof to Github. I used Edge browser and captured the network traffic. This is the response I got from the server:
![image](https://github.com/community/community/assets/91502725/e781e809-3b08-4653-88d9-1f3e83e4cd56)
It looks like the server is rejecting the upload because the payload exceeds the set size limit. I checked my file size and could verify that my uploaded file was roughly 736KB:
![image](https://github.com/community/community/assets/91502725/24f1ffbe-70c6-40c9-9092-91a7bfec5375)
This seems to be technical bug that Github should fix, but they ask to report technical bugs to the community forum...

Thank you for your attention!

**Reply by mattrmd** [reply]
Following @GitAceHub's lead, I was able to fix this issue on Linux using `convert <filename>.jpg -quality 75 <filename>_75.jpg`, which shrank my image to under 500kB. Since my image was previously 875kB, 500kB seems to be the real cutoff, not 1MB. Hope this helps.
